operators/merge_from_scene.py

```diff
@@ -0,0 +1,38 @@
+import bpy
+import operator
+
+
+class MergeFromSceneStrip(bpy.types.Operator):
+    """
+    Merge into the current scene meta data from a scene strip,
+    re-creating strips, actions, animation tracks and markers.
+    """
+    bl_idname = "power_sequencer.merge_from_scene_strip"
```

Comment:
You should start using the same format we have in the other operators for `bl_idname`, `bl_description` etc. Check this for example: https://github.com/GDquest/Blender-power-sequencer/blob/master/operators/add_speed.py. The `doc` variable is also important. This all ties in with auto-registering the key shortcuts & creating the docs for the repository. If the operator doesn't have a shortcut yet just leave `doc['shortcuts']` as an empty array, i.e. `[]`

Reply:
Awesome! I wasn't aware of the docs specs, thanks for the heads up. Gonna standardize everything for sure 📝 👍

Reply:
A quick note: this is something Razvan added a few days ago to automate docs, shortcuts generation, etc.